### DISCUSSION TITLE
ToonSquid is Better Than Procreate Dreams

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#455] [Show and tell]
[Is Procreate Dreams Better Than ToonSquid (Procreate Dreams Review)](https://youtu.be/0cYbmLnsRDc?si=YLCF9GxwMcnhqQcE) I got Procreate Dreams last night and managed to test it out. Compared to ToonSquid, they should have called this Procreate Nightmares.

**Comment by tunixx** [comment]
Yo he probado ambos y para el gran show que hicieron en procreate, pense y espera algo mejor, mas funciones, pero la verdad es que ToonSquid, con todas las limitaciones solo es superado por procreate en el "timelapse" record, como sketchbook, y que esto@procreate lo llaman "perform", es lo unico mejorado y toonsquid debe implementar. 
--------------------------
I have tried both and for the great show they did in Procreate, I thought and expected something better, more functions, but the truth is that ToonSquid, with all the limitations, is only surpassed by Procreate in the "timelapse" record, as a sketchbook, and They call this @procreate "perform", it is the only thing improved and toonsquid should implement.

**Comment by kavlor** [comment]
I don't understand it's need to limit the number of video tracks to one while Toonsquid has no limit. Also it doesn't appear to output transparent video like HEVC+Alpha unless I am missing something.

I do like it's keyable warpmesh  but if it can't output transparency then I have to mess about in something like Lumafusion to remove the plain background.

**Reply by amalelemal** [reply]
Because PD is only raster, so it sacrifices video to allow more raster layers. Similarly, in ToonSquid, as soon as you’re dealing with raster, there is a limitation based on the device capacity. My opinion only, of course.

**Comment by Guccisocks** [comment]
Thought it was rude af how some people in the procreate dream thread have basically said "good bye toonsquid all the best with all" like this app is/was less superior
Even though a whole team designed, coded and tested procreate dreams I'm surpised how lacking that first version is and how bad the general usability is. 
I think it's easy to be wowed by a feature list on paper but then overlook how powerful an app is thats designed with good UI and workflow is when you actually have to animate something. 
Toonsquid is still my fav animation app to use and I dont see that changing for a while

**Reply by amalelemal** [reply]
Agree. In fact, what might have actually happened is that due to massive publicity preceding the release of PD, and massive disappointment shortly thereafter, some of the crowd that got excited about animation, trickled down to ToonSquid, thus increasing the sales. I hope that in this way ToonSquid actually benefited from PD’s advertising budget :)

 Personally, I have both (for the price, it’s worth it), and I think what happened is that Savage needed to start receiving some revenue to be able to continue working on their ambitious feature list. I’ve seen some mentions that Procreate itself was a similar mess when it was first released, and got cleaned up in the following years.

Lastly, unless PD implements vector, ToonSquid shouldn’t have to worry about its niche, imho, as it has a powerful vector-based workflow.

**Reply by wandrinsheep** [reply]
I’m here for it and support them as long as it doesn’t take 10 years to get good…hopefully.

**Comment by ArtsyStar1** [comment]
Ikr? Procreate Dream is very incomplete and this is soooo disappointing! I hope that after the 2.0 update releases, I will reinstall Toonsquid again.

**Comment by Cr8iveMike** [comment]
I resonate with your disappointment in the missing features in Procreate Dreams. However, I do not agree with your perception of it being below Toonsquid or any other animation app on the iPad. I like toonsquid and also love Dreams but I believe both apps have their pros and cons respectively. When I hear you guys overly make complaints about the tools you expected to have come with it but didn't after so much buzz about it revolutionizing the animation industry. I wondered if you Have sat down to think that the veterans who tested the app beforehand are doing exceptional masterpieces with Dreams and not Toonsquid? Since TS came out, I haven't seen any grand breaking works down on it that could sell it more. I even advise the creator to initiate a kind of challenge every month to showcase the potential of TS as they did PD. Psychologically, people will be drawn to it, (I mean the veterans and the pros in the game like Aaron Blaise, Nikolai Lockertsen, and the works that came preinstalled. PD is new and not yet perfected but in due time with the measure of attention on it, it will take its grand stage in animation space. No doubt I like Toonsquid because of vector tweening and some others which to me place it above callipeg but then….

**Reply by wandrinsheep** [reply]
That’s terribly disingenuous. ToonSquid is hardly known and not pushed by Apple as much as procreate which is basically apples baby at this point so it will land with massive fanfare. Ofcourse you’re not gonna see big industry professionals talk about it when they hardly bother to check(outside a few).Procreate has name and brand recognition and anything that can be done so far in dreams barring filters can also be done in ToonSquid. The inverse is hardly as true. But I won’t lie having eternal undos and offstage snd limitless frames and optimization is where dreams currently shines. But that doesn’t augment its toolsets for creative works instead it  provides a solid base to start from with a great user experience, which is savages strong suit.

**Reply by DrGReaper** [reply]
I have used the vast majority of the major animation software programs out there dating back to the 90s.  Apps are fairly new to the game but I got hooked on Rough Animator just for fun with all of it's simplicity.  I was curious about Dreams so checked it out, it doesn't flow for me. They need some animators on their team it seems.  The good news is it forced me to keep looking into the apps out there and I found Squid. Exactly what I was hoping to find for quick on the go animation.
